azure-pipelines-ui-tests.dfe.yml

```diff
@@ -0,0 +1,220 @@
+schedules:
+- cron: 0 5 * * 1,2,3,4,5
+  branches:
+    include:
+    - dev
+  always: true
+- cron: 0 7 * * 1,2,3,4,5
+  branches:
+    include:
+    - dev
+  always: true
+- cron: 0 19 * * 1,2,3,4,5
+  branches:
+    include:
+    - refs/heads/dev
+  always: true
+name: $(date:yyyyMMdd)$(rev:.r)
+resources:
+  repositories:
+  - repository: self
+    type: git
+    ref: refs/heads/dev
+
+jobs:
+- job: 'Public'
+  displayName: Public UI tests
+  timeoutInMinutes: 160
+  cancelTimeoutInMinutes: 10
+  condition: succeededOrFailed()
+  pool:
+    vmImage: ubuntu-20.04
+  steps:
+  - checkout: self
+    clean: true
+    fetchDepth: 5
+    fetchTags: false
```

Comment:
I think maybe some of this kind of stuff -- here and elsewhere in the file -- can be removed? I wonder if this was included in the originally generated the yaml?